### ISSUE TITLE
Figure out an arm9 payload to restore backups

### ISSUE THREAD

**Issue by astronautlevel2** [astronautlevel2/StarUpdater#14]
In the event of upgrading to a bad commit (which just happened earlier today), an option to restore backups from an arm9 payload would be convenient.

**Comment by TheBaloneyboy** [comment]
This is possible, but something like BootCTR would have to be used to load the payload. The payload itself is very simple, search for a file entitled "arm9loaderhax.bin.bak" and rename it accordingly, but in order to use it, you would have to use a boot loader, or have that payload itself chain load Luma3DS.

**Comment by cheatfreak47** [comment]
@TheBaloneyboy That's true but in the unlikely event a Luma3DS release is broken, chances are the part that'd be broken wouldn't be payload chainloading so really that may be a precaution for the user to decide how they should handle.

This would still be a wise idea to make something like this.

**Comment by astronautlevel2** [comment]
I'm going to be taking another look at this soon(:tm:?)

**Comment by cheatfreak47** [comment]
Considering recent events, that'd be a good idea, although the chink in the stability armor here wasn't even Luma directly, if it happens once it could again.
